types/node-forge/index.d.ts

```diff
@@ -652,7 +686,18 @@ declare module 'node-forge' {
 
         function publicKeyToRSAPublicKey(publicKey: PublicKey): any;
 
-        type setRsaPublicKey = typeof rsa.setPublicKey;
+        function setRsaPublicKey(n: jsbn.BigInteger, e: jsbn.BigInteger): PublicKey;
```

Comment:
for this function and also for `setRsaPrivateKey` below, you might want to consider doing something like `const setRsaPublicKey: typeof rsa.setPublicKey`, just to avoid duplicating the types and making sure the function types are the same.

Reply:
I'll look into that now